README.md

```diff
@@ -84,18 +84,18 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        channel: [stable, beta, dev]
+        sdk: [stable, 2.7.2, 2.12.0-1.4.beta, beta, dev]
     steps:
       - uses: actions/checkout@v2
       - uses: dart-lang/setup-dart@v1
         with:
-          channel: ${{ matrix.channel }}
+          sdk: ${{ matrix.sdk }}
 
       - name: Install dependencies
-        run: dart pub get
+        run: pub get
```

Comment:
I changed this because when specifying a version, the `dart pub <command>` commands don't work, but `pub <command>` does. Let me know if I've missed something that should make this work the same way it does when specifying a channel like `stable` - which pulls in the latest release.

Reply:
This is...interesting. `dart pub get` is the new hotness...but it's only available in more recent SDKs. 🤷

Reply:
mmm... that could be it! I was basing it off of what happened when I tested w/ `2.7.2` - since that is what Workiva is currently stuck on.

When was `dart pub *` added @kevmoo? I'll change the example stable specific version to be that and restore the use of the `dart *` commands in the readme.

Reply:
mmm... looks like its only supported in latest stable... so maybe I'll change the example back and just add a callout notating that `dart pub` won't work when specifying versions before `2.12.0`.

Reply:
Added